src/utils/texts.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  FIELD_START_TIME: 'Start Time',
+  FIELD_END_TIME: 'End Time',
+
+  FIELD_DATE_DATE: 'Date',
+  FIELD_DATE_SELECT_DATE: 'Select a date',
+
```

Comment:
Be careful with this type of naming. It's the same problem of calling "backgroundBlue" in a theme. It's better to make clear the place where it will be used than refer to his content.

I would rather to have this called something like FIELD_ROOM_LABEL and FIELD_ROOM_PLACEHOLDER